docs/working-with-cdk-for-terraform/terraform-cloud.md

```diff
@@ -0,0 +1,13 @@
+# Connect Terraform Cloud with the CDK for Terraform
+
+This document describes two ways to connect Terraform Cloud with your CDK for Terraform codebase to continuously deploy your infrastructure.
+
+## Using the Terraform Cloud VCS Integration
+Terraform Cloud supports [connecting to VCS providers](https://www.terraform.io/docs/cloud/vcs/index.html). To use the VCS integration, commit the generated Terraform config (e.g. the `cdk.tf.json` and other files in `cdktf.out`) alongside your code so Terraform Cloud can use it to deploy your infrastructure.
```

Comment:
cdk.tf.json is in cdktf.out, so why not say cdktf.out?
Is there more config needed, e.g. pointing TFC to the tf.json file?

Reply:
Yes, you're right. Going to mention that.
Also going to adjust this to just `cdktf.out` 👍